### PR TITLE
Remove the Plone APIs conventions

### PR DESCRIPTION
**Pull request by thet**
They are moved to  http://docs.plone.org/develop/styleguide and http://docs.plone.org/develop/coredev/docs/git.html

Fixes: https://github.com/plone/plone.api/issues/294

### CODE REVIEW

File: docs/contribute/develop.rst [hunk 1]
```diff
@@ -181,3 +202,69 @@ This command also re-generates your documentation.
...
+
+        :param a: The first operand.
+        :type a: :class:`mod.path.A`
+
+        :param b: The second operand.
+        :type b: :class:`mod.path.B`
+
+        :rtype: int
+        :return: The sum of the operands.
+        :raise: `KeyError`, if the operands are not the correct type.
```
Comment (hvelarde) [inline docs/contribute/develop.rst]: it's `:raises:`

**Comment by hvelarde** [comment]
documentation is not code, so you must not break lines at column 80

**Comment by thet** [comment]
Tnx for reviewing! Done.

**Comment by hvelarde** [comment]
+1 for merging after a squash ;-)

**Comment by thet** [comment]
squash'd

**Pull request merged**